### PR TITLE
feat: filter session recordings by active time

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

So many session recordings, gotta watch 'em all doesn't work for everyone

## Changes

When using listing v3 let people choose to filter by total duration or active duration

## How did you test this code?

developer tests and seeing it working locally

### CODE REVIEW

**Comment by pauldambra** [comment]
a step for #15845

File: posthog/queries/session_recordings/session_recording_list_from_replay_summary.py [hunk 1]
```diff
@@ -175,6 +177,14 @@ def get_query(self) -> Tuple[str, Dict[str, Any]]:
...
         person_cte, person_cte_match_clause, person_person_cte_params = self._persons_cte_clause
 
+        session_is_active_clause = (
+            "AND active_time >= 0.44"
+            if self._filter.include_active_sessions_filter == "include"
+            else "AND active_time <= 0.13"
+            if self._filter.include_active_sessions_filter == "exclude"
+            else ""
+        )
+
```
Comment (benjackwhite) [inline posthog/queries/session_recordings/session_recording_list_from_replay_summary.py:187]: This is quite a bit different to how I would expect this feature to work.

I figured we wouldn't mark sessions as "active or inactive" but rather allow that when filtering on duration you can either filter for the "total duration" or the "active duration" and then specify your own thresholds.

File: frontend/src/scenes/session-recordings/playlist/SessionRecordingsPlaylistFilters.tsx [hunk 2]
```diff
@@ -124,6 +136,12 @@ export function SessionRecordingsPlaylistFilters({
                     <LemonLabel>Duration</LemonLabel>
                     {durationFilter}
                 </div>
+                {listingVersion === '3' ? (
+                    <div className="flex gap-2">
+                        <LemonLabel>Include</LemonLabel>
+                        {isActiveFilters}
+                    </div>
+                ) : null}
```
Comment (benjackwhite) [inline frontend/src/scenes/session-recordings/playlist/SessionRecordingsPlaylistFilters.tsx:144]: Personally I think this should be a select box for the `durationFilter` to choose between "active" and "total" duration. To me that would feel much clearer whilst also giving the most control to the user

**Review by benjackwhite — CHANGES_REQUESTED** [review]

File: posthog/queries/session_recordings/session_recording_list_from_replay_summary.py (hunk 1 above)
Comment (pauldambra) [reply posthog/queries/session_recordings/session_recording_list_from_replay_summary.py:187]: How do we avoid the space shuttle cockpit problem? 

If we hide the complexity here we can filter on click count, key press, active time, number of events etc etc with one control for people who don't care _how_ we're calculating activity. Whereas if we give multiple controls the user has control but some are overwhelmed with the number options....

Comment (benjackwhite) [reply posthog/queries/session_recordings/session_recording_list_from_replay_summary.py]: Definitely see that point. 

We have the same sort of issue I guess with the event filters, where most people simply want to do "show me recordings for this page" where we force them to find a pageview event, then expand filters, then search for current_url.... 🙈

I guess what I'm advocating for is - if the API is granular and offers all the options, we can craft the frontend to be "simple" or "advanced" whenever we want. I'd rather make it feel "simple" from the client and "powerful" from the API.

Comment (pauldambra) [reply posthog/queries/session_recordings/session_recording_list_from_replay_summary.py]: > I'd rather make it feel "simple" from the client and "powerful" from the API.

I want a book of Ben-isms, that's the second this week. I'm totally onboard with that 💖

**Review by benjackwhite — APPROVED** [review]
LOVE IT

**Pull request merged**